qiskit/quantum_info/states/statevector.py

```diff
@@ -359,6 +360,29 @@ def reverse_qargs(self):
         ret._op_shape = self._op_shape.reverse()
         return ret
 
+    def expectation_value_pauli(self, pauli):
```

Comment:
This should be an internal helper function
```suggestion
    def _expectation_value_pauli(self, pauli):
```